core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -2690,6 +2690,46 @@ public void updateMount(AlluxioURI alluxioPath, MountContext context)
     }
   }
 
+  /**
+   * Mounts an under storage path to an Alluxio path.
```

Comment:
The convention within Alluxio codebase is to not add javadocs to methods which are `@Override` methods.